CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKHitTester.java

```diff
@@ -456,4 +495,17 @@ public void queueRequest(final HitTestRequest request) {
         requestQueue.add(request);
         needsDisplayUpdate = true;
     }
+    
+    
+    // ========================> Helpers <======================== \\
+    static int counter = 0;
+    private void SaveToFile() {
```

Comment:
This isn't currently called right?  Can you move it to a section called ==> Debuggering <=== please?

Reply:
Moved to Debug section